config/permissions.yml

```diff
@@ -2299,6 +2299,11 @@ feature_statements_like_may_like:
     label: 'Stellungnahme als anonymer Nutzer liken (Aktion ausführen)'
     loginRequired: false
     parent: feature_statement
+feature_statements_location_and_document_refrence:
+    expose: true
+    label: 'Feature Stellungnahme mit Orts- und Dokumentbezug'
```

Comment:
I thought, we want slightly switch to englisch here

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/4120/commits/cebe7b364f5e45f4f6cd32750702a063045a568f